apps/src/blockly/eventHandlers.ts

```diff
@@ -139,6 +144,35 @@ export function storeWorkspaceWidth(e: GoogleBlockly.Events.Abstract) {
   }
 }
 
+export function setPathFill(e: GoogleBlockly.Events.Abstract) {
+  if (e.type === Blockly.Events.FINISHED_LOADING && e.workspaceId) {
+    const workspace = Blockly.Workspace.getById(
+      `${e.workspaceId}`
+    ) as ExtendedWorkspace;
+    let patternBlocks: ExtendedBlock[] = [];
+    patternBlocks = workspace
+      .getAllBlocks()
+      .map(block => block as ExtendedBlock)
+      .filter(block => block.getFillPattern());
+    patternBlocks.forEach(block => {
+      const pattern = block.getFillPattern();
+      if (pattern && block instanceof GoogleBlockly.BlockSvg) {
+        block.svgPathFill = Blockly.createSvgElement(
+          'path',
+          {class: 'blocklyPath'},
+          block.getSvgRoot()
+        );
+        block.svgPathFill.setAttribute('fill', 'url(#' + pattern + ')');
+        const pathDescription = block.pathObject.svgPath.getAttribute('d');
+        console.log({pathDescription});
```

Comment:
Did you mean to remove this `console.log`?

Reply:
Yes, will remove!